src/hiero_analytics/data_sources/github_ingest.py

```
@@ -529,12 +391,80 @@ def fetch_repo_issue_events_rest_since(
+) -> list[IssueTimelineEventRecord]:
+    """Fetch issue timeline events for one issue using GraphQL."""
+    query = load_query("issue_timeline")
+    since_value = since.astimezone(UTC).isoformat() if since is not None else None
+    cache_scope = f"{owner}_{repo}_{issue_number}"
+    cache_parameters = {
+        "owner": owner,
+        "repo": repo,
+        "issue_number": issue_number,
+        "since": since_value,
+    }
+    cached = load_records_cache(
```

Ruff (0.15.18)

[error] Undefined name `load_records_cache`

(F821)